src/server/service.rs

```diff
@@ -88,7 +88,8 @@ where
         Ok(Async::Ready(()))
     }
 
-    fn call(&mut self, req: Self::Request) -> Self::Future {
+    fn call(&mut self, mut req: Self::Request) -> Self::Future {
+        let _ = req.set_nodelay(true);
```

Comment:
@fafhrd91 it is been removed in c674ea912691d86379d610a4794a50cef4b2feac

Do you remember why? Or was it mistake?

Reply:
There is separate service for that

Reply:
You mean `StreamConfigurationService`?

Reply:
Well, this is the simplest solution I found to the issue. If you want to solve the issue in a different way, I could modify the pull request.

Reply:
Well, this is the simplest solution I found to the issue. If you want to solve the issue in a different way, I could modify the pull request.